native/NativeSampleApps/MobileSyncCompose/src/main/java/com/salesforce/samples/mobilesynccompose/app/MobileSyncComposeApp.kt

```diff
@@ -0,0 +1,16 @@
+package com.salesforce.samples.mobilesynccompose.app
```

Comment:
Add copyright in all the source files - you can copy them from elsewhere.

Reply:
Would the notice need to also go in files generated from the Jetpack Compose "new project" template provided by Android Studio?  Several were not modified like the `Colors.kt`.